src/Authenticator/CookieAuthenticator.php

```diff
@@ -224,7 +225,8 @@ protected function _createCookie($value): CookieInterface
             $data['path'],
             $data['domain'],
             $data['secure'],
-            $data['httpOnly']
+            $data['httponly'],
+            $data['samesite'],
```

Comment:
This is causing parse errors in PHP 7.2 builds.

```suggestion
            $data['samesite']
```